openlibrary/templates/batch_import.html

```diff
@@ -0,0 +1,37 @@
+$def with (batch_result=None)
+$# type: (BatchResult)
+
+<div class="batchImport">
+  <div id="contentBody">
+    <h1>$_('Batch Imports')</h1>
+    <p>$_('Attach a JSONL formatted document with import records.')</p>
+    <p>$:_('See the <a href=\"https://github.com/internetarchive/openlibrary-client/tree/master/olclient/schemata\">olclient import schemata</a> for more.')</p>
+    <form method="POST" enctype="multipart/form-data" action="">
+      <input type="file" name="batchImport" />
+      <br/>
+      <input type="submit" value="$_('Import')"/>
+    </form>
+
+    $if batch_result and batch_result.errors:
+      <p>$_('Import failed.')</p>
+      <p>$_('No import will be queued until *every* record validates successfully.')</p>
+      <p>$_('Please update the JSONL file and reload this page (there should be no need to reattach the file).')</p>
+      <p>$_('Validation errors:')</p>
+      $for error in batch_result.errors:
+        <li><strong>Line $error.line_number:</strong> $error.error_message</li>
+
+    $if batch_result and batch_result.batch:
+      <p>$_('Import results for batch:') $batch_result.batch.name</p>
```

Comment:
The batch ID is lightly what we'll use in future APIs
```suggestion
      <p>$_('Import results for batch:') Batch #$batch_result.batch.id ($batch_result.batch.name)</p>
```